src/components/views/styl/Application.styl

```diff
@@ -25,7 +25,7 @@ dt, dd {
     display: inline;
 }
 
-dt {
+#experiment-details dt {
```

Comment:
Are there other `dt`s that we don't want to style this way? I intentionally made this selector broad. The `dd` below, too.

If we want to make either/both of these selectors only apply to `#experiment-details`, we should move this into a different CSS file, too.

Reply:
Agreed with moving out of `Application`. This appears in the header and legend. The new legend has no need of the `::after`.

Reply:
Ah, I understand now. Thanks!